src/Surfnet/Stepup/Identity/Event/EmailVerifiedEvent.php

```diff
@@ -79,20 +74,18 @@ public function __construct(
         SecondFactorId $secondFactorId,
         SecondFactorType $secondFactorType,
         DateTime $registrationRequestedAt,
+        IdentifyingDataId $identifyingDataId,
         $registrationCode,
```

Comment:
What's reason for slotting `$identifyingDataId` in between `$registrationRequestedAt` and `$registrationCode` fields? To me it's logical to group the latter two. Same goes for all `*PossessionProvenEvent`s.